app/views/results/_result_asset.html.erb

```diff
@@ -18,7 +21,7 @@
       <%= image_tag 'medium/processing.gif' %>
     </span>
   <% else %>
-    <%= image_tag(preview_asset_path result.asset) if result.asset.is_image? %>
+    <%= image_tag result.asset.url(:medium) if result.asset.is_image? %>
```

Comment:
Actually we still use this method for non-downloadable images, but I think we can use simply this expression: asset.url(:medium) and remove preview method in controller at all